processor/src/operations/crypto_ops.rs

```diff
@@ -102,23 +102,21 @@ where
     /// - new value of the node, 4 elements.
     ///
     /// To perform the operation we do the following:
-    /// 1. Update the leaf node at the specified index in the advice provider with the specified
-    ///    root, and get the Merkle path to this leaf. If `copy` is set to true, we make a copy
-    ///    of the merkle set before updating it.
+    /// 1. Update the node at the specified index in the advice provider with the specified root,
+    ///    and get the Merkle path to it. It will make a copy of the Merkle tree before updating
+    ///    it.
```

Comment:
Let's change this to: "Update the node at the specified index in the Merkle tree with the specified root, and get the Merkle path to it."

Reply:
Fixed